www/docs/en/dev/cordova/storage/storage.md

```diff
@@ -162,10 +162,12 @@ IndexedDB provides a simple and easy to understand data model, much like LocalSt
 But unlike LocalStorage, you can create multiple databases, with multiple stores per
 database, and its asynchronous API and search indexes provide performance benefits.
 
-IndexedDB is supported by the underlying WebView on the following Cordova platforms:
+IndexedDB is supported by the underlying WebView on all platforms,
+with known limitations on web browser and Windows.
```

Comment:
web browser should be plural, shouldn't it?

Reply:
I am reluctant to make "browser" plural since I am referring to the Cordova `browser` platform and would rather refer to it by proper name, if possible.

Reply:
What about this idea:

```suggestion
with known limitations on the following platforms:

- browser
- Windows
```

Or do you have any other ideas?